mono/metadata/sysmath.c

```diff
@@ -204,7 +204,20 @@ ves_icall_System_Math_Ceiling (gdouble v)
 gint32
 ves_icall_System_Math_ILogB (gdouble x)
 {
-	return ilogb (x);
+	int ret;
+#if !HAVE_COMPATIBLE_ILOGB0
```

Comment:
Kinda makes me wonder why FP_ILOGB0 is not tested directly here without all the `autoconf` magic. Apparently it's a constant.

Reply:
hm.. makes sense indeed.